data/battle_scripts_1.s

```diff
@@ -7419,11 +7395,8 @@ BattleScript_AngerShellRet:
 	return
 
 BattleScript_WindPowerActivates::
-.if B_CHECK_IF_CHARGED_UP == TRUE
-	jumpifstatus3 BS_ATTACKER, STATUS3_CHARGED_UP, BattleScript_WindPowerActivates_Ret
-.endif
```

Comment:
More of a comment than a review, but wouldn't this change make `B_CHECK_IF_CHARGED_UP` do nothing?
I remember this config was added to address the fact that unlike other abilities, Wind Power in SV is "triggered" multiple times despite its effect doing nothing after the first time.

Reply:
I think that's because how Charge works in the games. Also the tests actually didn't pass with that config on I believe. I can remove that define if it does nothing atm.

Reply:
Yeah, thinking about it, that config was made for our own sake, not to be accurate to the games. You can remove it.

Reply:
Done, the config is removed